packages/class/src/runner.ts

```diff
@@ -7,20 +7,113 @@ import { CLASS } from "./class.js";
 import type { Config } from "./config.js";
 import { parse } from "./validate.js";
 
-export type ClassOutput = Record<string, number[]>;
+export interface OutputVariable {
+  key: string;
+  title: string;
+  unit: string;
+  symbol: string;
+}
+
+export const outputVariables: OutputVariable[] = [
+  {
+    key: "t",
+    title: "Time",
+    unit: "s",
+    symbol: "t",
+  },
+  {
+    key: "h",
+    title: "ABL height",
+    unit: "m",
+    symbol: "h",
+  },
+  {
+    key: "theta",
+    title: "Potential temperature",
+    unit: "K",
+    symbol: "θ",
+  },
+  {
+    key: "dtheta",
+    title: "Potential temperature jump",
+    unit: "K",
+    symbol: "Δθ",
+  },
+  {
+    key: "q",
+    title: "Specific humidity",
+    unit: "kg kg⁻¹",
+    symbol: "q",
+  },
+  {
+    key: "dq",
+    title: "Specific humidity jump",
+    unit: "kg kg⁻¹",
+    symbol: "Δq",
+  },
+  {
+    key: "dthetav",
+    title: "Virtual temperature jump at h",
+    unit: "K",
+    symbol: "Δθᵥ",
+  },
+  {
+    key: "we",
+    title: "Entrainment velocity",
+    unit: "m s⁻¹",
+    symbol: "wₑ",
+  },
+  {
+    key: "ws",
+    title: "Large-scale vertical velocity",
+    unit: "m s⁻¹",
+    symbol: "wₛ",
+  },
+  {
+    key: "wthetave",
+    title: "Entrainment virtual heat flux",
+    unit: "K m s⁻¹",
+    symbol: "(w'θ')ᵥₑ",
+  },
+  {
+    key: "wthetav",
+    title: "Surface virtual heat flux",
+    unit: "K m s⁻¹",
+    symbol: "(w'θ')ᵥ",
+  },
+];
+
+export type ClassOutput = {
+  [K in (typeof outputVariables)[number]["key"]]: number[];
```

Comment:
I think the idea of this is to have ClassOutput type be 

```ts
type ClassOutput = {
    t: number[];
    h: number[];
    theta: number[];
    dtheta: number[];
    q: number[];
    dq: number[];
    dthetav: number[];
    we: number[];
    ws: number[];
    wthetave: number[];
    wthetav: number[];
}
```

however it is

```ts
type ClassOutput = {
    [x: string]: number[];
}
```

I quickly tried to typing outputVariables with `as const satisfies readonly OutputVariable[]`, but that broke things downstream and did not have time to fix all those.